src/commands/new.ts

```diff
@@ -130,6 +151,10 @@ export default async (name: string, flags: Record<string, boolean>) => {
 		);
 
 		await editPackageJson(response.projectName, projectName);
+
+		if (response.packageManager === 'pnpm') {
```

Comment:
You also need to set https://pnpm.io/npmrc#public-hoist-pattern

```
public-hoist-pattern[]=@sapphire/*
```

Reply:
It's something I discovered since your previous PR btw

Reply:
Ok, will do it

Reply:
done